### DISCUSSION TITLE
AU Core Practitioner Consensus Discussion

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#37] [Consensus Discussions for R1]
Check out  [AU Core Practitioner](http://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-practitioner.html), noting the draft [general conformance](http://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html) and [Must Support](http://build.fhir.org/ig/hl7au/au-fhir-core/must-support.html) data obligations.

_To see the full constraints inherited from AU Base view the Key Elements Table_

AU Core Practitioner should support the minimum requirements to record, search, and fetch basic demographics and other administrative information about an individual practitioner.

Comment on:
- If this profile was mandated tomorrow, what problems would that cause for your organisation?
- What's missing?
- Should any other identifier besides HPI-I be supported in AU Core?
- Should anything change? What about the concepts you are using, is it sufficiently covered by the terminology in AU Core?

**Comment by heathfrankel** [comment]
Please allow active value of false to support cases when historical clinical resource reference a Practitioner that is no longer active within an organization

**Reply by nielmdrec** [reply]
At Medirecords we use Practitoner to represent Doctors (or Users of the System) which can be active or inactive.
So we also need support for Practitoner.active to be false

**Reply by dt-r** [reply]
Discussed in [2023-09-28 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587) and https://confluence.hl7.org/pages/viewpage.action?pageId=184930419.

Motion to remove constraints on Practitioner.active passed. Change tracked via https://github.com/hl7au/au-fhir-core/issues/60.

**Comment by dt-r** [comment]
From @ir4y

I have a question about the fixed value for the active field in Patient, Practitioner, PractitonerRole, and Organization.
Could you please share a bit more details on why it is a fixed true value?

There are multiple use cases when you need to make a resource inactive. @heathfrankel mentioned some of them in the GitHub discussion (please check related topics for dedicated resources) and all of them make sense to me.

_Originally posted by @ir4y in https://github.com/hl7au/au-fhir-core/discussions/43_

**Reply by dt-r** [reply]
Discussed in [2023-09-28 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587) and https://confluence.hl7.org/pages/viewpage.action?pageId=184930419.

Motion to remove constraints on Practitioner.active passed. Change tracked via https://github.com/hl7au/au-fhir-core/issues/60.

**Comment by heathfrankel** [comment]
I notice that AHPRA Regustration Number is represented in Practitioner.identifier and Practitioner.qualification.identifier. Should we have only one way of representing this? 
If we are going to associate this with the qualifications of the practitioner, then I would recommend using the qualification, however, I see this being an issue for existing systems as they only capture the AHPRA number and the associated qualification (and qualification code is mandatory).

I am not sure about the recording of address and telecom on the Practitioner itself, these are often (except perhaps mobile, although arguable) in the context of a particular role/location/organization. Practitioners may not like to disclose their private contact details.
I notice that PD2 IG does not include address or telecom on Practitioner.

**Reply by xtfer** [reply]
A Practitioner has an Ahpra number, and a Practitioner has Qualifications. Qualifications do not have Ahpra numbers.

The Ahpra Number is not an identifier for the Qualification and shouldn't be a part of that it. I can only assume it's here because that data is represented on the official AHPRA record.

**Reply by dt-r** [reply]
Discussed in [2023-09-28 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587). Outcome to raise AU Base tracker to raise issue against AU Base to improve the definition for Practitioner.qualification.identifier in AU Base, and the definition of how AHPRA is handled in AU Base: https://github.com/hl7au/au-fhir-base/issues/801

Agreement to review Practitioner.address and Practitioner.telecom against US Core and come back to group. For discussion on relevance to AU Core.

**Comment by heathfrankel** [comment]
Is it worth slicing official name use to ensure consistency for HI Service lookups etc?

**Reply by dt-r** [reply]
I am very nervous about slicing in AU Core; a discriminator has to work for all reasonable use of that element or it will block things it shouldn't. 

I'm not sure AU Core should go beyond implementation guidance when AU Core isn't what defines the HI Service lookup.

**Reply by dt-r** [reply]
Discussed in [2023-09-28 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=184920587). Outcome to raise AU Base tracker to consider capturing name used for lookup: https://github.com/hl7au/au-fhir-base/issues/802 and AU Core tracker to Develop guidance on use of Provider identifiers in Australia: https://github.com/hl7au/au-fhir-core/issues/55

**Comment by heathfrankel** [comment]
I suggest removing the mandatory requirement to support _revinclude of provenance until we at least have a profile for it.

**Reply by dt-r** [reply]
To be progressed via https://github.com/hl7au/au-fhir-core/issues/70.

**Comment by dt-r** [comment]
This discussion is now closed. Feedback on AU Core should be raised using GitHub issues. Feedback might be suggesting a feature, pointing out a place where a specification is unclear or over-restrictive, identifying a spelling or grammar issue, identifying a defect, or suggesting that an entire area needs to be rethought.

Integration with HL7 JIRA is coming and when available specification feedback will be migrated to JIRA. An announcement will be posted in this discussion when it is available.

Questions on this profile can be raised in chat.fhir.org: https://chat.fhir.org/#narrow/stream/179173-australia
